timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/rdf/Collection.java

```diff
@@ -79,6 +79,15 @@ public void add(Vertex entityVertex) {
     }
   }
 
+  public void add(Vertex entityVertex, Collection existingCollection) {
```

Comment:
this duplicates the other add. It would be better to pass the optional collection onwards